README.md

```diff
@@ -620,6 +621,22 @@ expect(submitButtons).toHaveLength(3) // expect 3 elements
 expect(submitButtons[0]).toBeInTheDOM()
 ```
 
+## `within` API
+
+Sometimes, there is no garauntee that the text, placeholder, or label you want to query is unique on the page. So you might want to explicity tell react-render-dom to get an element **only within** a particular section of the page. `within` is a helper function for this case.
```

Comment:
Small typo in here, `garauntee` should be `guarantee` :tada:

Reply:
What's react-render-dom?